connector/connect/common/src/main/protobuf/spark/connect/ml_common.proto

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
can we just merge ml_common.proto and ml.proto into a single file?

Reply:
No, we can't. Some imports may result in recursively importing.